### DISCUSSION TITLE
Future of vue3-openlayers

### DISCUSSION THREAD

**Discussion by 1Luc1** [MelihAltintas/vue3-openlayers#121] [General]
Are there any plans to actively maintain this project? Looking at the issues and commits it seems as though nothing is happening here.

This is a crosspost from [vue-leaflet](https://github.com/vue-leaflet/vue-leaflet/discussions/227), with the same topic.

**Comment by tmlmt** [comment]
It seems like a general observation regarding Vue implementations of cartography libraries. Can't find one which is well maintained and with most features I need. I am open to suggestions.

**Reply by 1Luc1** [reply]
maybe vue-leaflet gets back on track https://github.com/vue-leaflet/vue-leaflet/discussions/227#discussioncomment-4228223
and yesterday this repo was updated too

**Comment by d-koppenhagen** [comment]
@1Luc1 are you looking at a specific issue which is essential and where you see no progress?

**Reply by 1Luc1** [reply]
Thanks for your reply. No, not really. By the time I started this discussion, the last commit was on the Aug 8, 2022; so I just was kind of curious if this project is maintained or not.

**Reply by d-koppenhagen** [reply]
Okay. So in my opinion it is still maintained. Even if there are some month where no features / fixes have been tackled. Currently the project works with ol 6 and 7 and all dependencies are up-to-date.

**Reply by 1Luc1** [reply]
Again thank you for the information. I guess this the answer i was looking for :)
